README.md

```diff
@@ -0,0 +1,25 @@
+# LuaSQL 3.0 (proposed)
+http://www.keplerproject.org/luasql/
+https://github.com/keplerproject/luasql/
+
+LuaSQL is a simple interface from Lua to a DBMS. It enables a Lua program to:
+
+    * Connect to ODBC, ADO, Oracle, MySQL, SQLite and PostgreSQL databases;
+    * Execute arbitrary SQL statements;
+    * Retrieve results in a row-by-row cursor fashion.
+
+LuaSQL is free software and uses the same license as Lua 5.1. 
+
+Source code for LuaSQL can be downloaded from the Kepler Project Github page.
+
+- - -
+
+## LuaSQL 3.0.0 [09/Apr/2015]
+This is a proposed branch to add much needed features
```

Comment:
You're right, shouldn't have been there. I'll redo the pull req neater next week.